pom.xml

```diff
@@ -215,8 +215,8 @@
                     <artifactId>maven-compiler-plugin</artifactId>
                     <version>${version.plugin.compiler}</version>
                     <configuration>
-                        <source>${java.version}</source>
-                        <target>${java.version}</target>
+                        <source>1.8</source>
```

Comment:
It occurs to me that if you want to keep the `${java.version}` property, you could probably write a `maven-enforcer-plugin` rule (or [use a pre-existing one](https://maven.apache.org/enforcer/enforcer-rules/requireProperty.html)) to derive the javadoc API version from it, or vice versa.  Once we go to Java 9+ throughout, the various `1.x` flavors will go away and `9` will work everywhere.  I am not sure it's worth it now, so I support the hard coding you're doing here.